client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -53,10 +53,20 @@ function WPLineItem( {
 
 	return (
 		<div className={ joinClasses( [ className, 'checkout-line-item' ] ) }>
-			<LineItemTitle id={ itemSpanId } item={ item } />
+			<LineItemTitle id={ itemSpanId }>{ item.label }</LineItemTitle>
 			<span aria-labelledby={ itemSpanId }>
-				<LineItemPrice lineItem={ item } />
+				<LineItemPrice item={ item } />
 			</span>
+			{ item.sublabel && (
+				<LineItemMeta>
+					{ item.sublabel }
+					{ ( 'plan' !== item.type || ! shouldShowVariantSelector ) &&
+						': ' + translate( 'billed yearly' ) }
```

Comment:
This is... probably fine? But having the colon separate it does make me wonder how this will look in RTL languages. Maybe it would be better to have a translate string that interpolated the product name, the colon, and "billed yearly".